crates/oxc_ast/src/ast/js.rs

```diff
@@ -1,6 +1,6 @@
-// NB: `#[span]`, `#[scope(...)]`, `#[visit(...)]`, `#[visit_as(...)]` and `#[visit_args(...)]` do
-// not do anything to the code, They are purely markers for codegen used in
-// `tasts/ast_codegen` and `crates/oxc_traverse/scripts`. See docs in that crate.
+// NB: `#[span]`, `#[scope(...)]` and `#[visit(...)]` do NOT do anything to the code.
+// They are purely markers for codegen used in
+// `tests/ast_codegen` and `crates/oxc_traverse/scripts`. See docs in those crates.
```

Comment:
Oh dear! Shouldn't it be `tasks` not `tests`?

Reply:
Oh you are right, I don't know why have difficulties spelling this path😆 I'll take care of it.